src/main/java/Duke.java

```diff
@@ -1,10 +1,333 @@
+     */
+    private static int commandToDo(Task[] currentTasks, int numOfTasks, String splitUserInput) {
+        try {
+            if (splitUserInput.isBlank()) {
+                throw new DukeException("MISSING_DESCRIPTION");
+            }
+            currentTasks[numOfTasks] = new ToDo(splitUserInput);
+            printSeparator();
+            System.out.println("Got it. I've added this task: \n" + "  " + addBrackets(currentTasks[numOfTasks].getType())
+                    + addBrackets(currentTasks[numOfTasks].getStatusIcon()) + " " + splitUserInput + " \n" + "Now you have "
+                    + (numOfTasks + 1) + (numOfTasks == 0 ? " task" : " tasks") + " in the list");
+            numOfTasks++;
+            printSeparator();
+
+        } catch (DukeException error) {
+            printSeparator();
+            System.out.println(error);
+            printSeparator();
+        }
+        return numOfTasks;
+    }
+
+    /**
+     * Adds a Tasks.ToDo task to the currentTasks Tasks.Task array
+     *
+     * @param currentTasks Tasks Array of all tasks
+     * @param numOfTasks   Number of tasks
+     * @param userInput    User input
+     * @return numOfTasks Updated number of tasks
+     */
+    private static int commandEvent(Task[] currentTasks, int numOfTasks, String userInput) throws DukeException {
+        try {
+            /*If user input contains only one word*/
+            if (userInput.length() < 1) {
+                throw new DukeException("MISSING_DESCRIPTION");
+            }
+            /*Missing qualifier*/
+            if (!userInput.contains("/at")) {
+                throw new DukeException("MISSING_QUALIFIER");
+            }
+            String[] splitInfoAndDate = userInput.trim().split(EVENT_QUALIFIER, 2);
+            /*If user input has no description*/
+            if (splitInfoAndDate[0].isEmpty()) {
+                throw new DukeException("MISSING_DESCRIPTION");
+            }
+            /*If user input has no event date*/
+            if (splitInfoAndDate[1].isEmpty()) {
+                throw new DukeException("MISSING_INFO");
+            }
+            currentTasks[numOfTasks] = new Event(splitInfoAndDate[0], splitInfoAndDate[1]);
+            printSeparator();
+            System.out.println("Got it. I've added this task: \n" + "  " + addBrackets(currentTasks[numOfTasks].getType())
+                    + addBrackets(currentTasks[numOfTasks].getStatusIcon()) + " " + splitInfoAndDate[0] + " (at: " +
+                    splitInfoAndDate[1] + ")\n" + "Now you have " + (numOfTasks + 1) + (numOfTasks == 0 ? " task" : " tasks")
+                    + " in the list");
+            numOfTasks++;
+            printSeparator();
+        } catch (DukeException error) {
+            printSeparator();
+            //error.printStackTrace();
+            System.out.println(error);
+            printSeparator();
+        }
+        return numOfTasks;
+    }
+
+    /*Prints out a list of available commands*/
```

Comment:
This documentation is probably not necessary as the method name itself is self-explanatory. In general, if the method name itself is straightforward, it is okay to not document it :)

Reply:
Alright will update it in my next commit!